### ISSUE TITLE
In Tasks  allow empty Responsible field

### ISSUE THREAD

**Issue by ctplatectplate** [tine20/tine20#6889]
Tine version 2018.08.1: I can not add a task without a responsible, in the list on the left is there a favorite "Tasks without responsible"

**Comment by pschuele** [comment]
yup, i think, the validation of the field in the edit dialog should be removed.
